nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-components/src/main/java/org/apache/nifi/controller/StandardProcessorNode.java

```diff
@@ -2110,7 +2110,7 @@ private void migrateProperties(final Map<String, String> originalPropertyValues,
             final List<ControllerServiceCreationDetails> servicesCreated = propertyConfig.getCreatedServices();
             servicesCreated.forEach(serviceFactory::create);
 
-            overwriteProperties(propertyConfig.getProperties());
+            overwriteProperties(propertyConfig.getRawProperties());
```

Comment:
Thanks for the fix @krisztina-zsihovszki!
It looks good, however I don't know all the details around property migration.

One note here: `StandardControllerServiceNode` and `AbstractReportingTaskNode` use the same `getProperties()` method so it needs to be fixed there too.

Reply:
You are right, I changed those as well, thank you.